src/matrix/room/timeline/entries/reply.js

```diff
@@ -72,3 +75,14 @@ export function createReplyContent(entry, msgtype, body) {
     const newFormattedBody = formattedFallback + htmlEscape(body);
     return _createReplyContent(entry.id, msgtype, newBody, newFormattedBody);
 }
+
+function createThreadContent(entry, msgtype, body) {
+    return {
+        msgtype,
+        body,
+        "m.relates_to": {
+            "rel_type": "m.thread",
```

Comment:
In `isThread`, we use `io.element.thread`, but here we use `m.thread`. Should this be just one thing? Would it be good to extract it into a constant so that one can be switch into the other once (if) the threads MSC lands?